modules/custom/az_select_menu/src/Plugin/Block/AzSelectMenu.php

```diff
@@ -126,9 +126,12 @@ public function blockSubmit($form, FormStateInterface $form_state) {
   public function build() {
 
     $build = parent::build();
+    $menu_name = 'az-select-menu-' . $build['#menu_name'];
```

Comment:
You'll want to put this at the top of the file 
```suggestion
use Drupal\Component\Utility\Html;
    $menu_name = 'az-select-menu-' . $build['#menu_name'];
```